integration/go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/hyperledger/fabric-chaincode-evm/integration
+
+replace github.com/hyperledger/fabric-chaincode-evm/fab3 => ../fab3
+
+replace github.com/hyperledger/fabric-chaincode-evm/evmcc => ../evmcc
+
+require (
+	code.cloudfoundry.org/clock v0.0.0-20180518195852-02e53af36e6c // indirect
+	github.com/containerd/continuity v0.0.0-20181003075958-be9bd761db19 // indirect
+	github.com/fsouza/go-dockerclient v1.3.0
+	github.com/hyperledger/fabric v1.4.0
+	github.com/hyperledger/fabric-chaincode-evm/fab3 v0.0.0
+	github.com/onsi/ginkgo v1.10.2
+	github.com/onsi/gomega v1.7.0
+	github.com/tedsuo/ifrit v0.0.0-20191009134036-9a97d0632f00
+	github.com/willf/bitset v1.1.10 // indirect
+	go.etcd.io/etcd v3.3.17+incompatible // indirect
+)
+
+replace github.com/go-kit/kit => github.com/go-kit/kit v0.7.0
```

Comment:
should we force this to 0.8.0?

Reply:
Nope, we need to force to v0.7.0 because of fabric. We don't have a problem in evmcc and fab3 because we aren't building fabric in those situations and we don't have an compile issues there. In the integration tests we have to build Fabric 1.4.0 which is locked on v0.7.0 and v0.8.0 introduces compile errors.